app/controllers/api/v0/persons_controller.rb

```diff
@@ -33,6 +33,12 @@ def competitions
     render json: person.competitions
   end
 
+  def personal_records
```

Comment:
Shouldn't this be moved to a `users_controller`?

I would expect that routes for `persons` live in a `persons_controllers` and routes for `users` to live in a `users_controller`